README.md

```diff
@@ -1,68 +1,36 @@
-
-To learn React, check out the [React documentation](https://reactjs.org/).
-
-### Code Splitting
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/code-splitting
-
-### Analyzing the Bundle Size
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size
-
-### Making a Progressive Web App
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app
-
-### Advanced Configuration
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/advanced-configuration
-
-### Deployment
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/deployment
-
-### `npm run build` fails to minify
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
+# Som Timer
+Som Timer is a timer that cares. Based on the pomodoro technique™️, this application provides users with curated wellness content during break intervals. Build in rest, so you can focus best.
+
+[Deployed Front End](https://som-timer.herokuapp.com/) | [Deployed Back End](https://som-timer-be.herokuapp.com/)
+
+## Project in Action
+- GIFs coming soon!
+
+## Future Directions
+- Mood tracking: we'd like to incorporate mood tracking functionality, so that users can provide a sense of how they are feeling before & after their POM break. 
+  - Eventually, we'd like to use these ratings to help curate content for users
+
+## Installation Instructions
+1. Clone down the [Front End Repository](https://github.com/SOM-Timer/som_timer_fe)
+  - Run `git clone git@github.com:SOM-Timer/som_timer_fe.git <...>` in the terminal, where `<...>` is an optional way to customize the name of your local repo.
+2. Run `npm install` to install necessary dependencies
+3. Navigate to the root directory and run `npm run start` to view the app in action.
+
+## Tech Stack
+- React
+- React Hooks
+- SCSS
+- [Notifications API](https://developer.mozilla.org/en-US/docs/Web/API/Notifications_API)
+- Testing with:
+  - React Testing Library
+  - Jest
+
+## Contributors
+Front End Developers
+  - Aaron Burris-Deboskey: [GitHub](https://github.com/Abdeboskey) | [LinkedIn](https://www.linkedin.com/in/aaron-burris-deboskey/)
+  - Jake West: [GitHub](https://github.com/jkwest-93) | [LinkedIn](https://www.linkedin.com/in/jake-west-3840b71b4/)
+  - Rachel Williams: [GitHub](https://github.com/rwilliams659) | [LinkedIn](https://www.linkedin.com/in/rachelsarahwilliams/)
+Back End Developers 
+  - Chandler Hulstrom: [GitHub](https://github.com/Chulstro) | [LinkedIn](https://www.linkedin.com/in/chandler-hulstrom-34ab81150/)
+  - Dorion: [GitHub](https://github.com/sciencefixion)
+  - Sienna Kopf: [GitHub](https://github.com/sienna-kopf) | [LinkedIn](https://www.linkedin.com/in/sienna-kopf-b22b701b1/)
```

Comment:
aww our Linked In profiles too! Thanks for putting the time in to add that.

Reply:
Yes, nice touch Jake.